templates/artist/includes/artist_detail_investors.html

```diff
@@ -1,24 +1,89 @@
+{% load staticfiles %}
+{% load humanize %}
+{% load perdiem %}
+
 <div class="tabs-panel" id="investors">
-    <h2>INVESTORS</h2>
+    {% if user.is_authenticated %}
+    <h3>Invest in {{ campaign.project.reason }}</h3>
     <div class="break">
         <hr>
     </div>
+
+    <!-- Invest -->
+    {% if latest_campaign %}
```

Comment:
Agree. Left a note about adding the project artwork above the invest button. 

I think that would help clarify they are investing in that specific project and not the artist as a whole.